packages/babel-plugin-react-i18n/src/BabelPluginReactI18n.ts

```diff
@@ -0,0 +1,97 @@
+import {TemplateBuilder} from '@babel/template';
+import * as Types from '@babel/types';
+import {NodePath} from '@babel/traverse';
+import stringHash from 'string-hash';
+import path from 'path';
+import fs from 'fs';
+
+interface State {
+  importer(importArg: Types.ImportDeclaration): void;
+}
+
+const WITH_I18N_DECORATOR_NAME = 'withI18n';
+
+export default function injectWithI18nArguments({
+  types: t,
+  template,
+}: {
+  types: typeof Types;
+  template: TemplateBuilder<Types.ImportDeclaration | Types.ObjectExpression>;
+}) {
+  const fallbackTranslationsImport = template(
+    `import enTranslations from './translations/en.json';`,
+    {sourceType: 'module'},
+  );
+  function i18nParams({ID}) {
+    return template(
+      `withI18n({
+      id: '${ID}',
+      fallback: enTranslations,
+      async translations(locale) {
+          try {
+            const dictionary = await import(/* webpackChunkName: '${ID}-i18n' */ \`./translations/$\{locale}.json\`);
+            return dictionary;
+          } catch (err) {}
+        },
+    })`,
+      {
+        sourceType: 'module',
+        plugins: ['dynamicImport'],
+        preserveComments: true,
+      },
+    )();
+  }
+
+  return {
+    visitor: {
+      Program(_path: NodePath<Types.Program>, state: State) {
+        const lastImport = _path
+          .get('body')
+          .filter(subPath => subPath.isImportDeclaration())
+          .pop();
+        if (lastImport) {
+          state.importer = importDecl => lastImport.insertAfter(importDecl);
```

Comment:
It is very convenient that it is a local import and you can get away with just this and still put it in the right place. 👍Cool.